<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatMonitorSettingsPage</class>
 <widget class="QWidget" name="ChatMonitorSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Operation Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="operationMode">
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Operation modes:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;&quot;&gt;&lt;span style=&quot; font-weight:400; text-decoration: underline;&quot;&gt;Opt-In:&lt;/span&gt; &lt;span style=&quot; font-weight:400;&quot;&gt;Only buffers on the right side are shown in chatmonitor&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Opt-Out:&lt;/span&gt; Buffers on the right side will be ignored in chatmonitor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelAvailableBuffers">
         <property name="text">
          <string>Available:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BufferView" name="availableBuffers"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="deactivateBuffer">
         <property name="toolTip">
          <string>Move selected buffers to the left</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/16x16/actions/oxygen/16x16/actions/go-previous.png</normaloff>:/16x16/actions/oxygen/16x16/actions/go-previous.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="activateBuffer">
         <property name="toolTip">
          <string>Move selected buffers to the right</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/16x16/actions/oxygen/16x16/actions/go-next.png</normaloff>:/16x16/actions/oxygen/16x16/actions/go-next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelActiveBuffers">
         <property name="text">
          <string>Show:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BufferView" name="activeBuffers"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showHighlights">
     <property name="toolTip">
      <string>Show highlights in chatmonitor even if the originating buffer is ignored</string>
     </property>
     <property name="text">
      <string>Always show highlighted messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showOwnMessages">
     <property name="text">
      <string>Show own messages</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="alwaysOwn">
       <property name="toolTip">
        <string>Show own messages in chatmonitor even if the originating buffer is ignored</string>
       </property>
       <property name="text">
        <string>Include ignored buffers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showBacklog">
     <property name="toolTip">
      <string>Display messages from backlog on reconnect</string>
     </property>
     <property name="text">
      <string>Show messages from backlog</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="includeRead">
       <property name="toolTip">
        <string>Include read messages from backlog on reconnect</string>
       </property>
       <property name="text">
        <string>Include read messages</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BufferView</class>
   <extends>QTreeView</extends>
   <header>bufferview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>operationMode</tabstop>
  <tabstop>availableBuffers</tabstop>
  <tabstop>deactivateBuffer</tabstop>
  <tabstop>activateBuffer</tabstop>
  <tabstop>activeBuffers</tabstop>
  <tabstop>showHighlights</tabstop>
  <tabstop>showOwnMessages</tabstop>
  <tabstop>alwaysOwn</tabstop>
  <tabstop>showBacklog</tabstop>
  <tabstop>includeRead</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showOwnMessages</sender>
   <signal>toggled(bool)</signal>
   <receiver>alwaysOwn</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
  <connection>
   <sender>showBacklog</sender>
   <signal>toggled(bool)</signal>
   <receiver>includeRead</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
 </connections>
</ui>
